packages/core/src/App/Components/Layout/Header/account-info.jsx

```diff
@@ -48,24 +52,34 @@ const AccountInfo = ({
             }
             <Icon icon='IcChevronDownBold' className='acc-info__select-arrow' />
         </div>
-        <CSSTransition
-            in={is_dialog_on}
-            timeout={200}
-            classNames={{
-                enter    : 'acc-switcher__wrapper--enter',
-                enterDone: 'acc-switcher__wrapper--enter-done',
-                exit     : 'acc-switcher__wrapper--exit',
-            }}
-            unmountOnExit
-        >
-            <div className='acc-switcher__wrapper'>
-                <AccountSwitcher
-                    is_visible={is_dialog_on}
-                    toggle={toggleDialog}
-                    is_upgrade_enabled={is_upgrade_enabled}
-                />
-            </div>
-        </CSSTransition>
+        {is_mobile ?
```

Comment:
You can wrap this inside `MobileWrapper` from `@deriv/components` and remove `is_mobile`.

Reply:
This is a ternary operator condition, so I think we can use `isMobile` instead from `@deriv/shared`.

Reply:
I see, I meant something like this: 

```jsx
import {DesktopWrapper, MobileWrapper} from '@deriv/components';

<MobileWrapper>
            <AccountSwitcherMobile
                is_visible={is_dialog_on}
                disableApp={disableApp}
                enableApp={enableApp}
                toggle={toggleDialog}
                is_upgrade_enabled={is_upgrade_enabled}
            />
</MobileWrapper>
<DesktopWrapper>
            <CSSTransition
                in={is_dialog_on}
                timeout={200}
                classNames={{
                    enter    : 'acc-switcher__wrapper--enter',
                    enterDone: 'acc-switcher__wrapper--enter-done',
                    exit     : 'acc-switcher__wrapper--exit',
                }}
                unmountOnExit
            >
                <div className='acc-switcher__wrapper'>
                    <AccountSwitcher
                        is_visible={is_dialog_on}
                        toggle={toggleDialog}
                        is_upgrade_enabled={is_upgrade_enabled}
                    />
                </div>
            </CSSTransition>
</DesktopWrapper>
```